From 6ff005cf19363382fc867d7876a75fd8a386e894 Mon Sep 17 00:00:00 2001
From: Dave Prue <dave@prue.com>
Date: Thu, 31 Aug 2017 19:21:01 +0200
Subject: [PATCH] sunxi: Fix CONFIG_SUNXI_GMAC references

SUNXI_GMAC was still used to configure the code where as the
same has been renamed and moved to Kconfig in below commit
"sunxi: Move SUNXI_GMAC to Kconfig"
(sha1: 4d43d065db3262f9a9918ba72457bf36dfb8e0bb)

Signed-off-by: Dave Prue <dave@prue.com>
Reviewed-by: Simon Glass <sjg@chromium.org>
Reviewed-by: Jagan Teki <jagan@openedev.com>
Reviewed-by: Mark Kettenis <kettenis@openbsd.org>
Tested-by: Mark Kettenis <kettenis@openbsd.org>
[Tweek commit message, config_whitelist.txt, build-whitelist.sh]
Signed-off-by: Jagan Teki <jagan@openedev.com>
---
 arch/arm/include/asm/arch-sunxi/sys_proto.h | 2 +-
 board/sunxi/Makefile                        | 2 +-
 include/configs/sunxi-common.h              | 2 +-
 scripts/build-whitelist.sh                  | 4 ++--
 scripts/config_whitelist.txt                | 1 -
 5 files changed, 5 insertions(+), 6 deletions(-)

--- a/arch/arm/include/asm/arch-sunxi/sys_proto.h
+++ b/arch/arm/include/asm/arch-sunxi/sys_proto.h
@@ -24,7 +24,7 @@ void sdelay(unsigned long);
 void return_to_fel(uint32_t lr, uint32_t sp);
 
 /* Board / SoC level designware gmac init */
-#if !defined CONFIG_SPL_BUILD && defined CONFIG_SUNXI_GMAC
+#if !defined CONFIG_SPL_BUILD && defined CONFIG_SUN7I_GMAC
 void eth_init_board(void);
 #else
 static inline void eth_init_board(void) {}
--- a/board/sunxi/Makefile
+++ b/board/sunxi/Makefile
@@ -9,7 +9,7 @@
 # SPDX-License-Identifier:	GPL-2.0+
 #
 obj-y	+= board.o
-obj-$(CONFIG_SUNXI_GMAC)	+= gmac.o
+obj-$(CONFIG_SUN7I_GMAC)	+= gmac.o
 obj-$(CONFIG_SUNXI_AHCI)	+= ahci.o
 obj-$(CONFIG_MACH_SUN4I)	+= dram_sun4i_auto.o
 obj-$(CONFIG_MACH_SUN5I)	+= dram_sun5i_auto.o
--- a/include/configs/sunxi-common.h
+++ b/include/configs/sunxi-common.h
@@ -302,7 +302,7 @@ extern int soft_i2c_gpio_scl;
 #define CONFIG_PHYLIB
 #endif
 
-#ifdef CONFIG_SUNXI_GMAC
+#ifdef CONFIG_SUN7I_GMAC
 #define CONFIG_PHY_GIGE			/* GMAC can use gigabit PHY	*/
 #define CONFIG_PHY_ADDR		1
 #define CONFIG_MII			/* MII PHY management		*/
--- a/scripts/build-whitelist.sh
+++ b/scripts/build-whitelist.sh
@@ -13,10 +13,10 @@ export LC_ALL=C LC_COLLATE=C
 # There are two independent greps. The first pulls out the component parts
 # of CONFIG_SYS_EXTRA_OPTIONS. An example is:
 #
-#	SUNXI_GMAC,AHCI,SATAPWR=SUNXI_GPB(8)
+#	SUN7I_GMAC,AHCI,SATAPWR=SUNXI_GPB(8)
 #
 # We want this to produce:
-#	CONFIG_SUNXI_GMAC
+#	CONFIG_SUN7I_GMAC
 #	CONFIG_AHCI
 #	CONFIG_SATAPWR
 #
--- a/scripts/config_whitelist.txt
+++ b/scripts/config_whitelist.txt
@@ -2371,7 +2371,6 @@ CONFIG_STV0991_HZ_CLOCK
 CONFIG_ST_SMI
 CONFIG_SUNXI_AHCI
 CONFIG_SUNXI_EMAC
-CONFIG_SUNXI_GMAC
 CONFIG_SUNXI_GPIO
 CONFIG_SUNXI_MAX_FB_SIZE
 CONFIG_SUNXI_USB_PHYS
